packages/connector-wechat-native/src/index.test.ts

```diff
@@ -106,6 +106,17 @@ describe('getUserInfo', () => {
     });
   });
 
+  it('throws error if `openid` is missing', async () => {
+    nock(userInfoEndpointUrl.origin).get(userInfoEndpointUrl.pathname).query(parameters).reply(0, {
+      unionid: 'this_is_an_arbitrary_wechat_union_id',
+      headimgurl: 'https://github.com/images/error/octocat_happy.gif',
+      nickname: 'wechat bot',
```

Comment:
why don't these param names use camelCase?

Reply:
Followed the type definition of WeChat API response.

Reply:
ok